docs/developer/system-updates.md

```diff
@@ -0,0 +1,176 @@
+# System Updates: Publishing and Targeting Updates
+
+## Where to Add Updates
+
+All system updates are defined in a single shared UpdateRegistry widget file. To publish or modify updates, edit the following file:
+
+[instances/widgets.treasury-factory.near/widget/pages/settings/system-updates/UpdateRegistry.jsx](../../instances/widgets.treasury-factory.near/widget/pages/settings/system-updates/UpdateRegistry.jsx)
+
+> **Note:** All instances share this UpdateRegistry file. Updates published here will be visible to all instances unless using the `instances` property for targeting specific instances as described below.
+
+The system update functionality allows developers to publish updates that has to be applied and approved manually by treasury administrators (such as contract upgrades, policy changes, or instance app widget changes) to specific treasury instances or to all. This is managed through the `UpdateRegistry` widget and can be tested or previewed using the Playwright test suite.
+
+## How Updates Are Defined
+
+Updates are defined as JavaScript objects in the `UpdateRegistry` widget. Each update record can include the following fields:
+
+- `id` (number): Unique identifier for the update.
```

Comment:
Can it be any number? Is it actually used anywhere? Do we want to assign values in a numerical order?

Reply:
Yes we would assign these in numerical order. Updated the docs.